pkgs/applications/networking/irc/tiny/default.nix

```diff
@@ -3,8 +3,9 @@
 , rustPlatform
 , fetchFromGitHub
 , pkg-config
-, dbus
-, openssl
+, dbusSupport ? stdenv.isLinux, dbus
+, rustlsPreferredToOpenssl ? !stdenv.isLinux, openssl
+, desktopNotificationsSupport ? stdenv.isLinux
 , Foundation
```

Comment:
should be moved above the conditionals

Reply:
Done.  Also squashed and added a comment explaining that `useOpenSSL=false` does not disable TLS